Sorts/test/MergeSort.test.ts

```diff
@@ -0,0 +1,20 @@
+import { MergeSort } from "../MergeSort";
+
+/**
+ * Space complexity - O(n)
+ * Time complexity 
+ *      Best case   -   O(nlogn)
+ *      Worst case  -   O(nlogn)
+ *      Average case -  O(nlogn)
+ * 
+ * Merge Sort is a recursive algorithm and time complexity can be expressed as following recurrence relation. 
+ * T(n) = 2T(n/2) + O(n) 
+ * The solution of the above recurrence is O(nLogn).
+ */
+
+describe("Merge Sort", () => {
+    it("should return the correct value", () => {
```

Comment:
Consider generating random arrays and comparing against the builtin sort. A single test case definitely isn't enough. The description is useless.

Reply:
Generated array and compared with sorted array.